### ISSUE TITLE
The docs links show 404

### ISSUE THREAD

**Issue by Idimmusix** [trallnag/prometheus-fastapi-instrumentator#204]
The links to the github pages containing the docs show a 404.

**Comment by djordan-ri** [comment]
+1 Please fix.

**Comment by trallnag** [comment]
The page does exist anymore. What are you struggeling with?

**Comment by trallnag** [comment]
@djordan-ri, nope. The formerly existing documation was merely automatically generated from the source code and comments. And some point I removed this because one can just look directly at the code in GitHub.

I've collected some examples here: https://github.com/trallnag/prometheus-fastapi-instrumentator/tree/master/devel/examples

**Comment by djordan-ri** [comment]
Right. Understood. In that case, it might resolve the issue to link those examples in the top-level README. I hadn't found those exemplars, and I think most would be surprised to find them under `devel/`
